gems/aws-sdk-sqs/lib/aws-sdk-sqs/queue_poller.rb

```diff
@@ -349,20 +347,20 @@ def poll(options = {}, &block)
       # @param [Integer] seconds
       def change_message_visibility_timeout(message, seconds)
         @client.change_message_visibility({
-          queue_url: @queue_url,
-          receipt_handle: message.receipt_handle,
-          visibility_timeout: seconds,
-        })
+                                            queue_url: @queue_url,
```

Comment:
I know rubocop auto fixes these - but I think its more readable (and the style we use elsewhere in the V3 sdk) to keep the previous indention in these cases.

Reply:
I'll update this and other ones that I see that looks really out of place.

Reply:
I think rubocop is doing this because of the curly braces. If those are removed, auto formatting should work.